### DISCUSSION TITLE
Guide: How to Manage Your Supabase Database with Directus - Problems when enabling the PostGIS extension

### DISCUSSION THREAD

**Discussion by sebastianbachmann** [directus/directus#15133] [Archive]
Hello Everyone,

i am having trouble getting Directus and Supabase working together. I‘ve followed the steps within this Guide: https://directus.io/guides/directus-plus-supabase/ 

But when i want to start Directus with npx directus start those errore come up: https://gist.github.com/sebastianbachmann/5f957baae2a8f76741be104a30a4d0b5 Any ideas?

**Comment by azrikahar** [comment]
based on the `relation "geometry_columns" does not exist` error and quick lookup to land at this answer here: https://gis.stackexchange.com/a/227637, could it be when you enabled the PostGIS extension, it's not in the `public` schema?

I noticed the SQL command for enabling extensions described in their docs: https://supabase.com/docs/guides/database/extensions#enable-and-disable-extensions, mentions that it's created on the `extensions` schema instead of `public`. Perhaps you can try running `create extension postgis with schema public;` instead? (or just `create extension postgis;`)

**Reply by sebastianbachmann** [reply]
Need to check this the next week. Could be that i didn't added PostGIS extension to a public schema. I disabled the extension and created the project without it. Thanks and sorry for my late reply!
